dashboard/build/asset-manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
do we need to keep built files for the dashboard in git history?

Reply:
Yes. The reason is that when a user install singularity using go install ...@latest, they don't need to install npm for the dashboard, the compiled assets will be embeded inside singularity binary.
Some Singularity users complaint about difficulties to make npm working on server environment. I also recall Brenda mentioned same complaints for boost as well.

Reply:
Seems like there is a way to tell github not to show the diff for those files. At least it will make code review neater. WIll give it a try tmr

Reply:
added something in .gitattributes, seems to be working